modules/outbound-domain-event-handler/package.json

```diff
@@ -40,16 +36,32 @@
         "dep:update": "ncu -u"
```

Comment:
```suggestion
        "dep:update": "ncu -u",
        "audit:check": "yarn npm audit"
```

Reply:
Removed audit scripts in the modules as there is not yarn.lock files in the modules